tempodb/compactor.go

```diff
@@ -119,7 +81,7 @@ func (rw *readerWriter) doCompaction() {
 			level.Warn(rw.logger).Log("msg", "unable to find meta during compaction.  trying again on this block list", "err", err)
 		} else if err != nil {
 			level.Error(rw.logger).Log("msg", "error during compaction cycle", "err", err)
-			metricCompactionErrors.Inc()
+			metrics.MetricCompactionErrors.Inc()
```

Comment:
i can go either way on this change, but it is a pretty big shift. do you intend to move all tempodb metrics here eventually?